File: app/src/main/java/com/vn/ezlearn/modelresult/CategoryResult.java
package com.vn.ezlearn.modelresult;

import com.vn.ezlearn.models.ItemMenu;

import java.io.Serializable;
import java.util.List;

/**
 * Created by FRAMGIA\nguyen.duc.manh on 25/09/2017.
 */

public class CategoryResult implements Serializable {
    public Boolean success;
    public List<ItemMenu> data;
}

File: app/src/main/java/com/vn/ezlearn/modelresult/CategoryResult.kt
package com.vn.ezlearn.modelresult

import com.vn.ezlearn.models.Category

import java.io.Serializable

/**
 * Created by FRAMGIA\nguyen.duc.manh on 25/09/2017.
 */

class CategoryResult : Serializable {
    var success: Boolean = false
    var data: List<Category>? = null
}
